recipes/libsigcpp/2.x.x/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
```

Comment:
I would like to see the latest version available too: https://github.com/libsigcplusplus/libsigcplusplus/releases/tag/3.0.4

Reply:
3.x.x is already available. This PR specifically adds 2.x.x